examples/datalog-sleep.py

```diff
@@ -3,13 +3,14 @@
 import log
 
 # Log the temperature every 5 minutes
-@run_every(s=5)
+@run_every(min=5)
 def log_temperature():
     log.add(temp=temperature())
 
 while True:
+    # Display the temperature when button A is pressed
     if button_a.is_pressed():
-        # Display the temperature when button A is pressed
         display.scroll(temperature())
-    # To go sleep and wake up with run_every or button A
+    # To go sleep, wake up when button A is pressed, and ensure the
+    # function scheduled with run_every still executes in the background
     power.deep_sleep(wake_on=button_a, run_every=True)
```

Comment:
since `run_every=True` is the default, does it need to be explicitly specified here?

Reply:
No, it's no longer necessary, but I though it'd be good to keep it to be explicit and also so that is clear when reading the example code that the `run_every` function will continue to run while it sleeps.